### PR TITLE
GIX-2080: Enable tokens flag for sns-governance e2e test

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Enable tokens feature flat for mainnet.

In this PR, set flag to true for sns-governance test.

# Changes

* Set tokens flag to true in sns-governance.spec and change the flow to adapt to the tokens page.
* Add choosing the dropdown option in `getSnsTokens`. This worked before because it was in the same universe as the SNS and was initially selected.
* Add helper method `clickOnSnsUniverse` to SelectUniverseList.
* Add method to get SNS rows in TokensTable PO and helper constructor in TokensTableRow PO.

# Tests

Only test changes.

# Todos

- [ ] Add entry to changelog (if necessary).

Not necessary.

### CODE REVIEW

**Comment by lmuntaner** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #4248** <a href="https://app.graphite.dev/github/pr/dfinity/nns-dapp/4248?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/dfinity/nns-dapp/4248?utm_source=stack-comment).

**Comment by lmuntaner** [comment]
@dskloetd please review

File: frontend/src/tests/e2e/sns-governance.spec.ts [hunk 1]
```diff
@@ -1,45 +1,47 @@
...
-      .getMainAccountCardPo()
-      .getBalance()
-  ).toEqual("20.00");
+  expect(await snsUniverseRow.getBalanceNumber()).toEqual(askedAmount);
 
   step("Stake a neuron");
   await appPo.goToNeurons();
+
+  await appPo.openUniverses();
+  await appPo.getSelectUniverseListPo().clickOnSnsUniverse(snsProjectName);
```
Comment (dskloetd) [inline frontend/src/tests/e2e/sns-governance.spec.ts:43]: Why does clicking on the universe card as before no longer work?

Comment (lmuntaner) [reply frontend/src/tests/e2e/sns-governance.spec.ts:43]: Before it was getting the name from the selector by getting the first card, storing the name and clicking on it.

But now I get the name from the first SNS in the tokens table.

I don't want to rely that the first universe will be the same.

I could have also removed the check of the SNS in the accounts page and move to the neurons earlier. But that kind of changed the coveraga and I wanted to keep it as similar as possible.

Comment (dskloetd) [reply frontend/src/tests/e2e/sns-governance.spec.ts:43]: It wasn't really in the accounts page but rather in the side bar, which used to be the same in the accounts page and the neurons page. So to me this feels less similar. But it's fine with me.

**Review by dskloetd — APPROVED** [review]

File: frontend/src/tests/e2e/sns-governance.spec.ts (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/tests/e2e/sns-governance.spec.ts:43]: There is no sidebar in the tokens page though.

**Pull request merged**